spec/plugin.spec.js

```diff
@@ -221,4 +221,48 @@ describe('ManifestPlugin', function() {
       });
     });
   });
+
+  describe('assets added by third parties to "compilation.assets"', function() {
+    it('are included also when enabled.', function(done) {
+
+      var options = webpackConfig({
+        manifestOptions: {
+            includeCopiedAssets: true
+        }
+      });
+
+      // add plugin to get third party assets included
+      let Mock = require(path.join(__dirname, './plugin-mock'));
```

Comment:
`let Mock = require('./plugin-mock');` and can yoou put it at the very beginning of the file (Maybe find a better name ;) )